Composer/packages/client/src/shell/lgApi.ts

```diff
@@ -99,7 +99,7 @@ function createLgApi(
     addLgTemplate: updateLgTemplate,
     getLgTemplates,
     updateLgTemplate,
-    debouncedUpdateLgTemplate: debounce(updateLgTemplate, 250, { leading: true, trailing: true }),
+    debouncedUpdateLgTemplate: debounce(updateLgTemplate, 250),
```

Comment:
This (and the lu api) should be thoroughly tested. When changed in the past it caused some bugs with the editing experience.

Reply:
Make sense, i can help do more testing on this one. My expectation was it should be OK, since it's not removing denounce (which I remember we did in certain places and caused issues), it's just do the debounce in the default config. I will double check is the {leading: true, trailing: true} config was added for a specific reason, and will test whether it can be removed. Thanks.

Reply:
I will do more test here